src/javascripts/_helpers.js

```diff
@@ -91,12 +91,32 @@ var UK_Parliament = (function () {
 
       throw 'Sorry, there is no such ancestor with the identifiers you referenced: ' + JSON.stringify(attributes);
 
+    },
+
+    /**
+     * Toggle class to switch styles
+     * @param {object} element Element to toggle class on
+     * @param {string} active_state By default, this is set to 'active'
+     */
+    toggleClass = function(element, active_state) {
+
+      active_state = active_state || 'active';
+
+      if (element !== null) {
+        if (element.classList.contains(active_state)) {
```

Comment:
Could use the conditional (ternary) operator as a short cut to this if/else statement

Reply:
I think that would look ugly.